magento/magento2
# Removed

### Description
On frontend framework will update session expiration time on every request after session cookie is set.

### Fixed Issues (if relevant)
1. magento/magento2##7627: Cookie expiration time is not extended when browsing the frontend.

### Manual testing scenarios
1. Browse frontend until PHPSESSID is set. Take a look on PHPSESSID expiration time.
2. Open any page.
3. Make sure PHPSESSID expiration time is updated.

### Contribution checklist
